pkg/minikube/cluster/cluster.go

```diff
@@ -75,27 +76,27 @@ func StartHost(api libmachine.API, config MachineConfig) (*host.Host, error) {
 	glog.Infoln("Machine exists!")
 	h, err := api.Load(constants.MachineName)
 	if err != nil {
-		return nil, fmt.Errorf(
+		return nil, errors.Errorf(
```

Comment:
That's better, but if we're going to be strict (I'm not too bothered), then stuff in pkg/minikube/cluster/cluster.go should probably be unaware of the command calling it. Also, we should always try to use `os.Args[0]` in case someone has symlinked the command or renamed it for their own use, but again that should probably only be in cmd & subpackages.